release-notes/6.0/preview/api-diff/preview1/Asp.Net/6.0-preview1_Microsoft.AspNetCore.Http.Connections.md

```diff
@@ -0,0 +1,11 @@
+# Microsoft.AspNetCore.Http.Connections
+
+``` diff
+ namespace Microsoft.AspNetCore.Http.Connections {
+     public static class NegotiateProtocol {
+-        public static NegotiationResponse ParseResponse(Stream content);
```

Comment:
@BrennanConroy did we nuke this thing?

Reply:
Yes, it already was throwing and marked obsolete. Now it's gone.

Reply:
Does Azure SignalR use this?

Reply:
Not the sdk, and if they did it was broken after 2.2